apps/mudita-center-e2e/src/page-objects/mc-update-modal.page.ts

```diff
@@ -10,9 +10,25 @@ class ModalPage extends Page {
     return $('[data-testid="modal-title"]')
   }
 
-  get modalContentUpToDate() {
+  get updateNotAvailable() {
     return $('[data-testid="app-update-not-available"]')
   }
+
+  get updateAvailable() {
+    return $('[data-testid="app-update-available"]')
```

Comment:
I see there is app-update-available element. You didn't use it in `/specs/settings/mc-version-check-for-updates.e2e.ts:58`